standup/auth0/pipeline.py

```diff
@@ -0,0 +1,111 @@
+import logging
```

Comment:
This is the new stuff.

I broke up the callback view into a set of units which someone can assemble into an authentication pipeline depending on their needs. It's very very very loosely based on the one in python-social-auth.

Which begs the question "why not integrate this with python-social-auth?" I'm not sure we can because the bulk of the work for auth0 is done in the callback view and not in an auth backend.